docs(contributing): add details on Chromatic setup (#79)
Spendesk/grapes: CONTRIBUTING.md

### Chromatic

We use [Chromatic](https://www.chromatic.com) to check for visual bugs. In order to significantly reduce the number of screenshots taken, only one story per component named `Snapshot` is taken by Chromatic.

> This special story contain every variant and state of a specific component.

Reviewer: ```suggestion
This special story contains every variant and state of a specific component.
```

> For instance, if a component have 10 variants and 10 corresponding stories, everything is consolidate into a single story `Snapshot` in order to only take 1 screenshot instead of 10 separate screenshot for each variant.

Reviewer 2: missing chars:
```suggestion
For instance, if a component have 10 variants and 10 corresponding stories, everything is consolidated into a single story `Snapshot` in order to only take 1 screenshot instead of 10 separate screenshots for each variant.
```

The code for this special story `Snapshot` can be found here: https://github.com/Spendesk/grapes/blob/master/src/test-utils/SnapshotsContainer.tsx.

> However, this solution doesn't work this [interative stories](https://storybook.js.org/docs/writing-stories/play-function). In some cases, user interaction is required before a screenshot can be taken, such as selecting an option from a dropdown menu or opening a modal. For these types of stories, we kept the traditional approach of taking a screenshot per story.

Reviewer: ```suggestion
However, this solution doesn't work for [interative stories](https://storybook.js.org/docs/writing-stories/play-function). In some cases, user interaction is required before a screenshot can be taken, such as selecting an option from a dropdown menu or opening a modal. For these types of stories, we kept the traditional approach of taking a screenshot per story.
```
